### PR TITLE
update MLOS documentation

### PR DESCRIPTION
**Pull request by ImenBizid**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -348,6 +348,28 @@ WARNING: It is recommended to use an enabled-GPU node to run the deep learning t
...
+|If True,  the tasks will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
+| Boolean (default=True)
+| `Docker_IMAGE`
+| Specifies the docker image that will be used to execute the task.
+| String (default="activeeon/dlm3"
+| `TASK_ENABLED`
+| If False, the will be ignored, it will not be executed.
+| Boolean (default=True)
+| `LIMIT_OUTPUT_VIEW`
+| Specifies how many rows of the CSV will be previewed in the browser to check each task results.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: | Specifies how many rows of the CSV will be previewed in the browser to check each task results. 
Specifies how many rows of the data frame will be previewed in the browser to check each task results.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 
-.Add_Data_Task variables
+.Drop_Columns_Task variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `FILE_URL`
-| Put the URL of the data that you need to add.
+| `COLUMNS_NAME`
+| The list of columns that nedd to be dropped. columns names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: that nedd => that need

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 
-.Add_Label_Task variables
+.Encode_Data_Task variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `FILE_URL`
-| The URL of the file containing the labels that you need to concatenate.
+| `COLUMNS_NAME`
+| The list of columns that nedd to be encoded. columns names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: that nedd => that need

Comment (andrewssobral) [reply src/docs/MLOS/MLOSUserGuide.adoc]: columns names => Columns names

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 4]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 |===
 
-*How to use this task:* Could be used after <<Feature_Vector_Extractor>> and <<Import_Data>> tasks.
+NOTE: More details about the source code of this task can be found https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.filter.html[here^].
 
-NOTE: More details about the source code of this task can be found https://pandas.pydata.org/pandas-docs/stable/merging.html[here^].
+===== Merge_Data
 
-===== Filter_Data
+*Task Overview:* Merge DataFrame objects by performing a database-style join operation based on a specific referene column specified in the `REF_COLUMN` variable.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: referene => reference

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 5]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 
-*Task Overview:* Query the columns of your data with a boolean expression.
+*Task Variables:*
+
+.Merge_Data_Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `REF_COLUMN`
+| The list of columns to restrict to. columns names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: columns names => Columns names

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 6]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `QUERY`
-| The query string to evaluate.
+| `SCALER_NAME`
+| The list of columns to restrict to. columns names should be separated by a comma.
+| List [RobustScaler, StandardScaler] (default=RobustScaler)
+| `COLUMNS_NAME`
+| The list of columns that will be scaled. column names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: columns names => Columns names

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 7]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
 
-.Filter_Data_Task variables
+.Scale_Data_Task variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `QUERY`
-| The query string to evaluate.
+| `SCALER_NAME`
+| The list of columns to restrict to. columns names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: columns names => Columns names

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 8]
```diff
@@ -588,91 +648,70 @@ NOTE: More details about the source code of this task can be found https://panda
...
+*Task Overview:* Calculate the histogram of a dataframe based on a reference column that need to be specified in the variable `REF_COLUMN`.
+
+*Task Variables:*
+
+.Summarize_Data_Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `GLOBAL_MODEL_TYPE`
+| The model that will be used to summarize grouped by data.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: | The model that will be used to summarize grouped by data. 
 The model that will be used to summarize the data.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 9]
```diff
@@ -588,91 +648,70 @@ NOTE: More details about the source code of this task can be found https://panda
...
 |===
 
-*How to use this task:* Could be connected with <<Add_Label>> if you need to train a model using supervised learning algorithms or with <<Train_Clustering_Model>> if you need to train a model using unsupervised machine learning techniques.
+NOTE: More details about the source code of this task can be found https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram.html[here^].
 
-===== Time_Series_Feature_Extractor
+===== Tsfresh_Features_Extraction
 
-*Task Overview:* Extract features by considering the temporal distribution of data.
+*Task Overview:* Calculate a comprehensive number of time series features based on the library TSFRESH.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: Suggestion: add a link to the library

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 10]
```diff
@@ -499,75 +518,116 @@ image::pattern_file.png[align="center"]
...
+*Task Overview:* Subset columns of a dataframe according to the specified list of coluumns in  the `COLUMNS_NAME` variable.
+
+*Task Variables:*
+
+.Filter_Columns_Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `COLUMNS_NAME`
+| The list of columns to restrict to. column names should be separated by a comma.
```
Comment (andrewssobral) [inline src/docs/MLOS/MLOSUserGuide.adoc]: column names => Column names

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 9 above)
Comment (ImenBizid) [reply src/docs/MLOS/MLOSUserGuide.adoc]: For the link of the TSFRESH library it already exists in the Note sub section. For all the other comments, they are fixed now. Thanks for the review 👍

**Pull request merged**